selenoid.go

```diff
@@ -623,7 +645,7 @@ func reverseProxy(hostFn func(sess *session.Session) string, status string) func
 					r.URL.Scheme = "http"
 					r.URL.Host = hostFn(sess)
 					r.URL.Path = remainingPath
```

Comment:
Here for BiDi protocol I don't need to modify `r.URL.Path`, but I don't want to write a lot of code with `if else` statements (there is no ternary operator in the go). So I just modify `remainigPath` in code above, which also used in logger (nex line of code).